suite-common/graph/src/graphDataFetching.ts

```diff
@@ -222,11 +222,20 @@ export const getMultipleAccountBalanceHistoryWithFiat = async ({
                 identity,
                 endOfTimeFrameDate,
                 forceRefetch,
-            }).then(balanceHistory => ({
-                coin,
-                descriptor,
-                balanceHistory,
-            })),
+            })
+                .then(balanceHistory => ({
+                    coin,
+                    descriptor,
+                    balanceHistory,
+                }))
+                .catch(error => {
+                    console.error(
+                        `Unable to fetch GRAPH balance history for ${coin} account:`,
+                        error,
+                    );
+                    error.message = `${coin.toLocaleUpperCase()}: ${error.message}`;
```

Comment:
Does `toLocaleUpperCase` instead of `toUpperCase` bring something? Are we sure it wont produce different results on some locales?

Reply:
That's mistake fixed.